### ISSUE TITLE
Count Complaints and Bounces separately 

### ISSUE THREAD

**Issue by benstaf** [knadh/listmonk#1877]
I'm always frustrated when I see bounces and complaints lumped together

I want to see complaints counted separately because AWS SES requires to manage them separately, complaints are much more costly than bounces

**Comment by MaximilianKohler** [comment]
They are separate and you can control them separately in the settings under "bounces".
![Screenshot 2024-05-29 082748](https://github.com/knadh/listmonk/assets/3606996/5e05bd15-d60f-4bd5-9754-5ddd37aa4dd3)

**Comment by benstaf** [comment]
![Screenshot_2024-05-29-19-27-50-259_com android chrome-edit](https://github.com/knadh/listmonk/assets/8183572/42286268-341a-4457-8734-7581db95e7db)
![Screenshot_2024-05-29-19-29-41-260_com android chrome-edit](https://github.com/knadh/listmonk/assets/8183572/57a6e73a-a857-4f50-8adc-430b6836f3b1)


I meant the statistics metrics are incomplete: no complaint metric, and no soft bounce metric 

That's the 2 metrics that need to be monitored in AWS SES (and probably elsewhere)

**Comment by MaximilianKohler** [comment]
Yes, listmonk doesn't have a specific metric for complaints, but AWS does. AWS doesn't count soft bounces, only hard ones.

**Comment by MaximilianKohler** [comment]
I think you can get the exact count in Cloudwatch.

**Comment by benstaf** [comment]
Inmho, it would be better to have Hard Bounces and complaints on the campaign dashboard.

They are important deliverability metrics

**Comment by MaximilianKohler** [comment]
> Inmho, it would be better to have Hard Bounces and complaints on the campaign dashboard.
> 
> They are important deliverability metrics

I agree. If no one else is interested in implementing this I'll take a look and see if it's something I can do. It might be too cluttered to separate "hard", "soft" and "complaint" on the Campaigns page, but "bounce" and "complaint" possibly.

For that reason, it may be better to only have it on the analytics page. That may be beyond my expertise.

I wonder what Knadh thinks before starting on this.

**Comment by benstaf** [comment]
That would be nice to be able to customize which metrics to show on the campaign dashboard:

I personally don't need to see clicks and bounces

On the other hand, hard bounces and complaints are the two deliverability metrics that are important to monitor on a campaign basis (must pause the campaign immediately if they skyrocket)

**Comment by MaximilianKohler** [comment]
You seem to know more about coding than I do. Wouldn't it be as simple as copy-pasting the "bounces" section on the Analytics page and changing the "bounce" detection to "complaints"?

But perhaps a better solution would be to use the same graph but different colors for "soft", "hard", "complaint". That would be more difficult for me to do.

**Comment by cbleek** [comment]
I sent a mail with Listmonk via Amazon SES using a long existing account. There was a sudden increase in my bounce rate. I paused the sending manually and restarted it hours later. I want Listmonk to monitor the Amazon SES metrics and adjust the sending. 

Am I in the right place?

Here is a screenshot of the Amazon Metrics

<img width="519" alt="screen2" src="https://github.com/user-attachments/assets/8c81461d-4313-4ff8-bcbc-2dae2dd73c4d">
